1func.py

```diff
@@ -0,0 +1,10 @@
+list1=[1,2,3,4,4]
+def f(list):
+    set_a = set(list1)
+    max = 0
+    for x in set_a:
```

Comment:
Can you please simplify or avoid for loop by using Python-specific features. Try to use list generators.